### ISSUE TITLE
[BUG] Compile error using woocommerce css

### ISSUE THREAD

**Issue by justinkruit** [bootscore/bootscore#801]
### Describe the bug

The following errors happen when trying to compile the Bootscore WooCommerce scss.

```
Warning: Array to string conversion in F:\LocalWP\Local Sites\bootscore\app\public\wp-content\themes\bootscore-main\inc\scssphp\src\Compiler.php on line 927

Warning: Array to string conversion in F:\LocalWP\Local Sites\bootscore\app\public\wp-content\themes\bootscore-main\inc\scssphp\src\Compiler.php on line 927

Warning: Array to string conversion in F:\LocalWP\Local Sites\bootscore\app\public\wp-content\themes\bootscore-main\inc\scssphp\src\Compiler.php on line 927
```

Seems to be related to this: https://github.com/scssphp/scssphp/issues/606

### Steps to reproduce

Compile the scss file while including `bootscore-woocommerce`.

**Comment by crftwrk** [comment]
What is here exactly wrong? Compiles well and no errors?

**Comment by justinkruit** [comment]
They're warnings which happen with each compile. According to the issue on scssphp it happens when a specific selector is being used.

**Comment by crftwrk** [comment]
It's this selector:

https://github.com/bootscore/bootscore/blob/2382c5292568053c2538b36ea5487dc1605554a7/assets/scss/bootscore-woocommerce/_wc-qty-btn.scss#L27

cc @darrinbreen, @mswirydowicz

**Comment by finalwebsites** [comment]
I get this warnings with every pageview and I have the idea that my site takes 5 seconds to load :(
![Screenshot 2024-09-02 om 12 28 10](https://github.com/user-attachments/assets/bc758546-9d61-4887-a921-3dd1262e47e5)

**Comment by justinkruit** [comment]
@finalwebsites We are working on a fix (potentially #829). The higher loading times is because Bootstrap gets compiled on load. You can either turn off dev modus on WordPress, or add the following code to your child theme:
```
add_filter('bootscore/scss/skip_environment_check', '__return_true');
```

**Comment by finalwebsites** [comment]
@justinkruit thanks a lot!
